x/btcstaking/types/validate_parsed_message.go

```diff
@@ -206,6 +201,6 @@ func ValidateParsedMessageAgainstTheParams(
 	return &ParamsValidationResult{
 		StakingOutputIdx:   stakingOutputIdx,
 		UnbondingOutputIdx: 0, // unbonding output always has only 1 output
-		MinUnbondingTime:   minUnbondingTime,
+		MinUnbondingTime:   parameters.MinUnbondingTimeBlocks,
```

Comment:
as we no longer calculate valid `MinUnbondingTime` in this functions, we can probably remove this field from `ParamsValidationResult` . (the caller has access to ` parameters.MinUnbondingTimeBlocks` eitherway)

Reply:
Good point. Fixed in [6b4abc7](https://github.com/babylonlabs-io/babylon/pull/278/commits/6b4abc761e3966366f68f87982ab0c678132003d)